wiki/People/Tournament_Committee/en.md

```diff
@@ -26,7 +33,111 @@ Their responsibilities include:
+
+### Top Three Badge Support
+
+If a tournament host believes their tournament has achieved the highest level of production quality and organisation possible, they can request to receive badge prizes for all three finalists in their tournament. If a host requests to be awarded with this distinction, then in addition to the typical steps for tournament review as listed above, the assigned members will also notify the rest of the group of the request for top three badge support. Based on discussion around the tournament and its quality, either the request will be denied, or an anonymous vote will be called for the tournament to receive top three badge support.
+
+### Public Communication
+
+Public communication by the Tournament Committee is handled through several channels. The purposes for these sometimes overlap, but all are official sources of information kept up-to-date by the Tournament Committee.
+
+The Tournament Committee maintains an [account on X](https://twitter.com/osu_tcomm) (formerly Twitter) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
+
+The [Community Tournament Status Tracker](https://docs.google.com/spreadsheets/d/1bV5MyrJZI0F52Bx9EcYxdfRu8qGnhWRBKdXHK9uPbO0/edit?usp=sharing) is a publicly viewable spreadsheet detailing the status of all tournaments that have requested official support, i.e. all tournaments that requested screening. This sheet shows all tournaments dating back to January 14th, 2023, and contains a description of the status of the tournament at any given time, including whether a badge was applied or denied. If a badge was denied, the reason for denial is stated on the sheet.
+
+The [Official Tournament Support Updates thread](/community/forums/topics/1715676) is a pinned thread in the [Tournaments & Contests forum](/community/forums/126) which contains all updates to the Official Tournament Support wiki dating back to February 1st, 2023. This thread is updated and bumped every time a change is made to the Official Tournament Support wiki. This thread also contains links to the Official Support wiki and the Community Tournament Status Tracker.
+
+### Adding New Members
+
+New members to the Tournament Committee must meet the following basic criteria before being considered for the position at all:
+- Be at least 18 years old
+- Pass a security check
```

Comment:
```suggestion
```

Reply:
Security check was discussed with Chillier as being usable in the docs.

Reply:
Fine by me then

Reply:
i think it'd be worth clarifying that it's a check for recent account infringements, as that's the wording used elsewhere instead of a vague "security check"